packages/react-composites/tests/browser/chat/ChatComposite.test.ts

```diff
@@ -63,21 +70,56 @@ test.describe('Chat Composite E2E Tests', () => {
     await browser.close();
   });
 
-  test('composite loads correctly with participant list', async () => {
-    await pages[0].bringToFront();
-    await pages[0].waitForSelector(dataUiId(IDS.sendboxTextfield));
-    await pages[0].waitForSelector(dataUiId(IDS.participantList));
-    stubTimestamps(pages[0]);
-    expect(await pages[0].screenshot()).toMatchSnapshot('1-chat-screen.png');
+  test('composite pages load completely', async () => {
+    for (const idx in pages) {
+      await compositeLoaded(pages[idx]);
+      expect(await pages[idx].screenshot()).toMatchSnapshot(`1-page-${idx}-chat-screen.png`);
+    }
   });
 
   test('can send message', async () => {
+    const page = pages[0];
+    await page.bringToFront();
+    await page.type(dataUiId(IDS.sendboxTextfield), 'How the turn tables');
+    await page.keyboard.press('Enter');
+    await page.waitForSelector(`[data-ui-status="delivered"]`);
+    stubTimestamps(page);
```

Comment:
what does `stubTimestamps` do?

Reply:
Changes all the timestamps on a page to `timestamps` otherwise visual parity tests will always fail because messages will have different timestamps

Reply:
Recommend renaming this function so that it conveys the intent vs. what it does.

How about `stubNonDeterministicElements(page)` or `replaceNonDeterministicElementsWithPlaceholder(page)`?

Reply:
Too verbose.
I think `stubMessageTimestamps` should suffice.